sqtom/tests/test_fitting_1d.py

```diff
@@ -15,8 +15,45 @@
 
 import pytest
 import numpy as np
+from thewalrus.samples import photon_number_sampler
 from sqtom.forward_solver import degenerate_pmf
-from sqtom.fitting_1d import fit_1d, threshold_1d, marginal_calcs_1d
+from sqtom.fitting_1d import (
+    two_schmidt_mode_guess,
+    marginal_calcs_1d,
+    gen_hist_1d,
+    threshold_1d,
+    fit_1d,
+)
+
+
+@pytest.mark.parametrize("sq_0", [0.1, 1.0, 2.0])
+@pytest.mark.parametrize("eta", [0.1, 0.5, 1.0])
+def test_gen_hist_1d(sq_0, eta):
```

Comment:
Maybe we can reduce the number of tests, from 9 to perhaps 4 by eliminating one entry from each list?

Reply:
Sure. But the test only runs when someone pushes to github and not as a regular thing?